plugins/catalog-backend-module-gitlab/src/providers/GitlabOrgDiscoveryEntityProvider.ts

```diff
@@ -240,46 +392,42 @@ export class GitlabOrgDiscoveryEntityProvider implements EntityProvider {
     };
 
     for await (const user of users) {
-      if (!this.config.userPattern.test(user.email ?? user.username ?? '')) {
-        continue;
-      }
-
-      res.scanned++;
+      userRes.scanned++;
 
-      if (user.state !== 'active') {
+      if (!this.shouldProcessUser(user)) {
+        logger.debug(`Skipped user: ${user.username}`);
         continue;
       }
 
       idMappedUser[user.id] = user;
-      res.matches.push(user);
+      userRes.matches.push(user);
     }
 
     for await (const group of groups) {
-      if (!this.config.groupPattern.test(group.full_path ?? '')) {
-        continue;
-      }
+      groupRes.scanned++;
 
-      if (
-        this.config.group &&
-        !group.full_path.startsWith(`${this.config.group}/`)
-      ) {
+      if (!this.shouldProcessGroup(group)) {
+        logger.debug(`Skipped group: ${group.full_path}`);
         continue;
       }
+      logger.debug(`Processed group: ${group.full_path}`);
 
-      groupRes.scanned++;
       groupRes.matches.push(group);
 
       let groupUsers: PagedResponse<GitLabUser> = { items: [] };
       try {
-        groupUsers = await client.getGroupMembers(group.full_path, ['DIRECT']);
+        groupUsers = await client.getGroupMembers(group.full_path, [
+          'DIRECT',
+          'INHERITED',
```

Comment:
same comment as `all_available: true` but for `INHERITED` 😄

Reply:
Done, please let me know your thoughts! :)